app/assets/javascripts/discourse/app/templates/components/search-result-entry.hbs

```diff
@@ -0,0 +1,66 @@
+<div class="fps-result">
+  <div class="author">
+    <a href={{result.userPath}} data-user-card={{result.username}}>
+      {{avatar result imageSize="large"}}
+    </a>
+  </div>
+
+  <div class="fps-topic">
+    <div class="topic">
+      {{#if bulkSelectEnabled}}
+        {{track-selected selectedList=selected selectedId=result.topic class="bulk-select"}}
+      {{/if}}
+
+      <a href={{result.url}} {{action "logClick" result.topic_id}} class="search-link">
+        {{raw "topic-status" topic=result.topic showPrivateMessageIcon=true}}
+        <span class="topic-title">
+          {{#if result.useTopicTitleHeadline}}
+            {{html-safe result.topicTitleHeadline}}
+          {{else}}
+            {{#highlight-search highlight=q}}
+              {{html-safe result.topic.fancyTitle}}
+            {{/highlight-search}}
+          {{/if}}
+        </span>
+      </a>
+
+      <div class="search-category">
+        {{#if result.topic.category.parentCategory}}
+          {{category-link result.topic.category.parentCategory}}
+        {{/if}}
+        {{category-link result.topic.category hideParent=true}}
+        {{#if result.topic}}
+          {{discourse-tags result.topic}}
+        {{/if}}
+        {{plugin-outlet name="full-page-search-category" args=(hash result=result)}}
+      </div>
+    </div>
+
+    <div class="blurb container">
+      <span class="date">
+        {{format-date result.created_at format="tiny"}}
+        {{#if result.blurb}}
+          -
```

Comment:
wrap this in a span please

Reply:
separator ?

Reply:
you only need a class if you think that's going to be styled

Reply:
I used 'delimiter' but happy to switch it to separator.

Reply:
ok I've switched it in favour of your suggestion.